src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -59,6 +59,10 @@
  * {@link CNNVariantWriteTensors} and {@link CNNVariantTrain}.
  * CNNVariantTrain will create a json architecture file and an hd5 weights file, which you can use with this tool.
  *
+ * The advanced argument `info-annotation-keys` is available for models trained with different sets info field annotations.
+ * In order to do this you must first train your own model with the tools {@link CNNVariantWriteTensors} and {@link CNNVariantTrain}.
+ * Otherwise, providing this argument with anything but the standard set of annotations will result in an error.
```

Comment:
Is there any way to make the Python error more informative?  `ValueError: Error when checking input: expected annotations to have shape (7,) but got array with shape (6,)` from #5939 isn't so clear.

Reply:
I added an additional warning on the java side, it's a bit tricky to change the error on the python side without a big refactor.